{"id": "121687", "title": "Bug 121687: Exception throws when exporting data in a table within a grid cell", "description": "", "OB": "When attempting to export data in a table that is inside a grid cell, an exception is thrown in the ReportEngineService.getMetaData() function.", "EB": "When exporting data in two tables, the expected result is that the data should be exported without any exceptions.", "SR": "To reproduce the issue, follow these steps: 1. Insert one table into a grid cell. 2. Insert another table into another grid cell."}